plugins/linux-v4l2/v4l2-input.c

```diff
@@ -381,20 +376,30 @@ static void v4l2_device_list(obs_property_t *prop, obs_data_t *settings)
 		if (dp->d_type == DT_DIR)
 			continue;
 
-		dstr_resize(&device, 5);
-		dstr_cat(&device, dp->d_name);
+		char *dev_path = dp->d_name;
+		char buf[1024];
+		ssize_t len;
+		if ((len = readlink(dp->d_name, buf, sizeof(buf) - 1)) != -1) {
```

Comment:
I'm not sure I understand why we need to read the link here?

Reply:
I have no idea. I just borrowed this code from #3437.
Since you're the author and also don't know, I'll remove that. :)

Reply:
In my PR this is to find the real path to the device, but in your case you are joining the fake path to the link which normally would point off to no where, but in your case was working.